packages/instantsearch-react/docs/package.json

```diff
@@ -6,6 +6,7 @@
   },
   "devDependencies": {
     "babel-cli": "^6.11.4",
+    "babel-plugin-transform-object-rest-spread": "^6.8.0",
```

Comment:
I think we're already depending on the parent .eslintrc, so depending on the parent .babelrc makes sense.